src/debugger/Stepper.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+import type { StoppedData } from "./types.js";
+import { IsStatement } from "./../methods/is.js";
+import { BabelNode } from "babel-types";
+import invariant from "./../invariant.js";
+
+export class Stepper {
+  _prevStopData: StoppedData;
```

Comment:
This name is not true anymore: there may be other stops in the middle. Just call it "_stepStartData"